features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/TestConsole/TestConsole.jsx

```diff
@@ -425,14 +429,18 @@ class TestConsole extends React.Component {
                 </Typography>
             );
         }
-
-        if (api == null || swagger == null) {
+        if (!api || !swagger || !settings) {
             return <Progress />;
         }
-
         let authorizationHeader = api.authorizationHeader ? api.authorizationHeader : 'Authorization';
         authorizationHeader = 'testkey';
-        swagger.servers = settings;
+        if (swagger.openapi != null) {
```

Comment:
shall we use `!==` here, BTW I think this should catch from the eslint, Is eslint disabled?

Reply:
I fixed the issue. I enabled ESlint. but didn't show this issue.